fp.js

```diff
@@ -18,17 +23,20 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person) {
-  if(person.age < 35){
+people.filter(function(person) {
+  if(people.age < 35){
     peopleYoungerThan35.push(person)
   }
+  console.log(peopleYoungerThan35)
 });
-
+// naysayers.filter(function(naysayer){
+//   return naysayer.age > 300;
+// })
 // Convert the following to use `.forEach`
 
 var people = [
   {name: "Jane", age: 32},
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-var peopleWhoseNamesHaveTheLetterS = people.filter(function(person){return person.name.match("s")});
+var peopleWhoseNamesHaveTheLetterS = people.forEach(function(person){return person.name.match("s")});
```

Comment:
`forEach` has no return, or well returns `undefined` (no output, you can only achieve side effects with forEach), but `map` does have a return/output. if you replaced `.forEach()` with `map` your example would work. You have to declare an array above the forEach and then push each item to it.